src/components/filter-fields/SearchWithAutocomplete.jsx

```diff
@@ -32,6 +32,10 @@ const SearchWithAutocomplete = (props) => {
 
   useEffect(() => {
     setDomReady(true);
+    return () => {
+      setData([{}]);
+      setOffsetParams({});
+    };
   }, []);
```

Comment:
Do you change the state on destroying of the component?

Reply:
delete it